inst/examples/dashboard/style.scss

```
@@ -9,8 +13,116 @@
   --AppsilonPurple-weak-2: #BB87BE;
   --AppsilonRose-weak-1: #FF8692;
   --AppsilonYellow-weak-1: #FDE16E;
+}*/
+
+:root {
+  --AppsilonBlue:     #0099F9;
+  --AppsilonNavi:     #15354A;
+  --AppsilonIce:      #A6D5E0;
+  --AppsilonMint:     #1AFFD5;
+  --AppsilonYellow:   #FCD73D;
+  --AppsilonRose:     #FF364A;
+  --AppsilonPurple:   #770F7C;
+  --AppsilonGray:     #6E757B;
+  --AppsilonLightGray:#8A8E91;
+  --AppsilonAsh:      #B3B8BA;
+  --AppsilonSnow:     #FFFFFF;
+
+  --AppsilonBlue-weak-1:   #4CB8FB;
+  --AppsilonBlue-weak-2:   #80CCFC;
+  --AppsilonBlue-weak-3:   #CCEBFE;
+  --AppsilonBlue-strong-1: #0085E1;
+  --AppsilonBlue-strong-2: #0071CD;
+  --AppsilonBlue-strong-3: #005DB9;
```

Let's not align values like this; it's painful to maintain: one name change might require adjustment in many others which both takes time and increases the diff (not nice for the reviewer).

```suggestion
  --AppsilonBlue-weak-1: #4CB8FB;
  --AppsilonBlue-weak-2: #80CCFC;
  --AppsilonBlue-weak-3: #CCEBFE;
  --AppsilonBlue-strong-1: #0085E1;
  --AppsilonBlue-strong-2: #0071CD;
  --AppsilonBlue-strong-3: #005DB9;
```